rdr_service/resource/generators/consent_metrics.py

```diff
@@ -400,21 +402,30 @@ def _initialize_error_list():
         return error_dict
 
     @staticmethod
-    def send_consent_error_email(subject, body):
+    def send_consent_error_email(subject, body, recipients=None):
         """
         Send an email to the generic address that will trigger creation of a PTSC Service Desk ticket from the
-        email content.  NOTE:  Currently disabled/not implemented pending review of use of Sendgrid for emails
-        containing participant_id values
+        email content.
+        :param subject:  A string in the expected format (agreed upon w/PTSC) summarizing the error condition
+        :param body: Text/string (multi-line/multi-paragraph format agreed upon w/PTSC ) with details of each instance
+                     of the detected error condition
+        :param recipients: Destination email address list, if overriding the default config item
         """
 
-        # TODO:  Update config to add to/from email addresses for these reports, retrieve values from config
-        # email_obj = email_service.Email(subject, list([recipient_email]),  from_email, body)
-        # email_service.EmailService.send_email(email_obj)
-
-        # Temporary:  Log a warning while email code is disabled/not implemented.
-        error_count = body.count('Error Detected')
-        msg = '\n'.join([f'{subject} error detected for {error_count} consent file(s)',
-                         'Please manually generate the error report using consent-error-report tool'])
+        recipients = recipients or config.getSettingList(config.PTSC_SERVICE_DESK_EMAIL)
+        if recipients is None:
+            logging.error(f'No recipient address list available for consent error email generation')
```

Comment:
:nit would you mind removing the f-string here for the codacy issue

Reply:
Yes, just saw this (had changed the message content, forgot to change iftfrom an f-string).   Pushed up a new commit